src/mo_values/prim.ml

```diff
@@ -285,6 +285,23 @@ let prim trap =
     k (Blob (String.of_seq (Seq.map (fun v ->
       Char.chr (Nat8.to_int (Value.as_nat8 !(Value.as_mut v)))
     ) (Array.to_seq (Value.as_array v)))))
+  (* returns a single nat8 at the given index *)
+  | "blobGet" -> fun _ v k ->
```

Comment:
This would be better implemented using an overloaded array indexing construct:
``` Motoko
> ("\00":Blob)[0];
stdin:3.2-3.12: type error [M0075], expected array type, but expression produces type
  Blob
```

Reply:
We should make a decision over at https://github.com/dfinity/motoko/issues/966 first. If we merge Blob and Array types somehow, we might not need these prims?

Otoh, it's be easy to keep them around for backwards compat, so maybe ok?

Reply:
we can always remove prims if we want without notice

Reply:
Yes, I think it's fine to just a function for this at first. I'm not even sure I want to overload array subscripting, though its arguably already overloaded for [var T] and [T]. A good choice of library name can come later. `Blob.nth`, `Blob.sub`, `Blob.index`?